staff/francesc-destruels/cv/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+
+<body>
+    <header>
+        <h1>Curriculum Vitae</h1>
+    </header>
+    <nav>
+        <a href="#Basics">Basics</a>
+        <a href="#Formation">Formation</a>
+        <a href="#Profesional">Profesionals</a>
+        <a href="#Lenguages">Lenguages</a>
+        <a href="#About">About</a>
+    </nav>
+    <section>
+        <article>
+            <h3 id="Basics">Basics</h3>
+            <b>Full Name:</b> Francesc Destruels Rossello<br>
```

Comment:
`<b>` is not a semantic tag 😄

Reply:
More info in the following [link](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em)

Reply:
I will change it to <strong>